### ISSUE TITLE
Peripheral lung neovascularity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6567]
**Preferred term label:** Peripheral lung neovascularity

**Synonyms**

**Definition (free text, please give PubMed ID)**
A presence of small, tortuous, micronodular, serpiginous intrapulmonary vessels often in the subpleural lung or in proximity to centrilobular arterioles, coursing in directions inconsistent with known arteriolar anatomy; PMID: 16267251

**Parent term (use hpo.jax.org/app)**
Abnormality of the pulmonary vasculature HP:0004930

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
72% of PAH-CHD (56% with Eisenmenger's syndrome), 
22% of IPAH patients; PMID: 22205444
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik we would need to have a label such as Subpleural neovascularization -- please advise

**Comment by eswietlik** [comment]
Discussed with the radiologist, settled for 'Peripheral lung neovascularity', advised given to remove synonym  'Sheehan's vessels' as apparently never used

**Comment by pnrobinson** [comment]
added
